### PR TITLE
Convert to new override syntax

### PR DESCRIPTION
**Pull request by shr-project**

### CODE REVIEW

**Review by kraj — APPROVED** [review]
lgtm

**Comment by agherzan** [comment]
@shr-project do you mind merging the two commits on syntax change?

**Comment by shr-project** [comment]
> @shr-project do you mind merging the two commits on syntax change?

you mean squashing them? Please no as now the first script is created only by the script and the 2nd small is manually created and keeping them separate would help reviewing changes if the first one ever needs to be re-created with updated script.

**Comment by agherzan** [comment]
That makes sense @shr-project. Let's merge this. CI will keep failing until meta-oe catches up.

File: conf/layer.conf [hunk 1]
```diff
@@ -9,7 +9,7 @@ BBFILE_COLLECTIONS += "raspberrypi"
 BBFILE_PATTERN_raspberrypi := "^${LAYERDIR}/"
 BBFILE_PRIORITY_raspberrypi = "9"
 
-LAYERSERIES_COMPAT_raspberrypi = "hardknott honister"
```
Comment (agherzan) [inline conf/layer.conf:12]: So in this case it is not an override?

File: conf/layer.conf [hunk 2]
```diff
@@ -36,5 +36,5 @@ BBFILES_DYNAMIC += " \
     multimedia-layer:${LAYERDIR}/dynamic-layers/multimedia-layer/*/*/*.bbappend \
 "
 
-DEFAULT_TEST_SUITES_remove_rpi = "parselogs"
-DEFAULT_TEST_SUITES_append_rpi = " parselogs_rpi"
+DEFAULT_TEST_SUITES:remove:rpi = "parselogs"
+DEFAULT_TEST_SUITES:append:rpi = " parselogs:rpi"
```
Comment (agherzan) [inline conf/layer.conf:40]: How does this one work?

Comment (agherzan) [reply conf/layer.conf:40]: I think the testcase is called "parselogs_rpi".

File: conf/layer.conf (hunk 1 above)
Comment (agherzan) [reply conf/layer.conf:12]: re: LAYERSERIES_COMPAT

**Review by agherzan — CHANGES_REQUESTED** [review]
Added 2 comments.

File: conf/layer.conf (hunk 2 above)
Comment (shr-project) [reply conf/layer.conf:40]: will fix

File: conf/layer.conf (hunk 1 above)
Comment (shr-project) [reply conf/layer.conf:12]: no, here "raspberrypi" is the name of the layer, not OVERRIDE.

Comment (agherzan) [reply conf/layer.conf:12]: So a collection is not an override? I always thought it is.

**Review by agherzan — APPROVED** [review]
Looks good now. I think we can merge as CI will still fail until meta-oe catches up.

**Pull request merged**